e2e/utils/runtime-utils.ts

```diff
@@ -0,0 +1,19 @@
+import { Page } from 'puppeteer';
+import Button from 'app/element/button';
+
+const Selector = {
+  runtimeInitializerCreateButton: '//*[data-test-id="runtime-initializer-create")]'
+};
+
+export async function initializeRuntimeIfModalPresented(page: Page): Promise<void> {
+  let initializeButton: Button;
+  try {
+    initializeButton = new Button(page, Selector.runtimeInitializerCreateButton);
```

Comment:
`new Button` only initialize an object of Button. It won't throw exception. An action has to be performed. How about adding `await initializeButton`.exists()`?

Reply:
Oops, thanks - I want this to wait up to a minute, since the modal won't show up immediately. How about moving `waitForEnabled()` up to here? From looking at the code I thought this may work.

Reply:
Yes, it will work.